aircraft/cost.py

```diff
@@ -0,0 +1,112 @@
+"""Aircraft cost models"""
+from gpkit.shortcuts import Var, Model
+import gpkit
+gpkit.disable_units()
+
+
+class DAPCA4Cost(Model):
+    """
+    Cost Model from Raymer, Aircraft Design: A Conceptual Approach 3rd Edition,
+    pp 713-714
+
+    Note: this is also one instance of the RAND Corp cost models.
+
+    The total cost is Research, Development, Test and Evaluation Cost
+
+    Cost is in 2012 dollars
+
+    Engineering hours considers all airframe, propulsion and avionics
+    integration and design.  It does not include engineering of avionics,
+    propulsion, tooling or production planning.
+
+    Tooling hours covers all the preparation for production.  It also covers
+    ongoing tooling cost through production.
+
+    Manufacturing hours is the direct labor to fabricate the aircraft. It also
+    includes cost of labor of subcontractors.
+
+    Quality Control hours covers all inspections.
+
+    Flight test cost covers all cost to prove airworthiness.
```

Comment:
@mjburton11, I'm thinking some of these detailed definitions don't belong in the docstring. They're great, but I think they should be moved to either

1. The `label` string of the associated variable, or
2. An inline comment near the definition of the associated variable, if they are too long to be a reasonable `label`.